Cypress/cypress/pages/workflow/workflowValidation.js

```diff
@@ -54,8 +46,26 @@ Cypress.Commands.add("validateVerdict", (workflowName, agentName, expectedVerdic
     );
     cy.get('table')
         .find(`[data-cy=${workflowName}]`)
-        .find("[data-cy=ExperimentsPassed]")
-        .should("have.text", `Experiments Passed : ${ExperimentsPassed}/${TotalExperiments}`);
+        .then(($div) => {
+            cy.wrap($div)
+                .find("td")
+                .eq(3)
+                .should("have.text", agent); // Matching Target Agent
+            cy.wrap($div)
+                .find("[data-cy=ResScore]")
+                .should("have.text", `Overall RR : ${RScore}%`);
+            cy.wrap($div)
+                .find("[data-cy=ExperimentsPassed]")
+                .should("have.text", `Experiments Passed : ${ExperimentsPassed}/${TotalExperiments}`);
+            cy.wrap($div)
+                .find("[data-cy=WorkflowStatus]")
+                .should("have.text", expectedVerdict);
+            cy.wrap($div)
+                .find("td")
+                .eq(2)
+                .should("have.text", workflowName) // Matching Workflow Name

```

Comment:
This function should be only used for validating details. We can click on workflow name from outside of this function